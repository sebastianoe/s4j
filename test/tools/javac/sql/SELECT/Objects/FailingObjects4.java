/*
 * @test  (c) SO
 * @summary Fail due to wrong base class: product table needs to be queried or storage would need to be renamed
 * @compile businessobjects/Product.java
 * @compile businessobjects/Storage.java
 * @compile/fail/ref=FailingObjects4.out FailingObjects4.java -sqlrepo /Users/sebastian/ma/jdk7u-langtools/test/tools/javac/sql/repo.json
 */

import businessobjects.Product;
import businessobjects.Storage;

public class FailingObjects4 {
    public static void main(String[] args) {
    	Product p = new Product();
    	p.setStorageList(SQL[SELECT * FROM storage]);
    }
}