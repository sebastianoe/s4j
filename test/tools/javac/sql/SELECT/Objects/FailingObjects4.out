error: When using an SQL query in an ambiguous situation (esp. in potentially overloaded method calls), you need to specify an expected type hint.
1 error